spec/features/users/user_access_spec.rb

```diff
@@ -1,15 +1,17 @@
 require 'rails_helper'
 
-describe 'User Access', js: true do
+describe 'User Access' do
   context 'when Earthdata Login is required' do
-    context 'when a logged out user tries to view an access restricted page' do
-      before do
-        require_urs_login
+    before do
+      require_urs_login
+    end
 
+    context 'when a logged out user tries to view an access restricted page', js: true do
+      before do
         visit search_path
       end
 
-      it 'redirects them to URS for login' do
+      it 'redirects them to URS to login' do
         expect(page).to have_content('EARTHDATA LOGIN')
```

Comment:
So this test still confuses me why it works for Earthdata Login but not Launchpad, but logging `page.current_path` says it is on `/oauth/authorize`, and the screenshot indicates it seems to work:
![capybara-201806061814306141569212](https://user-images.githubusercontent.com/1174907/41068045-d57ae570-69b5-11e8-90e0-6e85cdc6e552.png)